SixTrack/sixtrack.s

```diff
@@ -3883,50 +3883,66 @@
           pi=4d0*atan(1d0)
 +ei
         crabfreq=ek(ix)*c1e3
-
+ 
         do j=1,napx ! loop over particles
-         crabamp=(ed(ix)/ejfv(j))*c1e3                                   !hr03
+         crabamp=ed(ix)                                  !hr03
+
 
 +if .not.tilt
 +if crlibm
-        yv(xory,j)=yv(xory,j) - crabamp*                                &!hr03
-     &sin_rn((((sigmv(j)/clight)*crabfreq)*2d0)*pi + crabph(ix))         !hr03
-      dpsv(j)=dpsv(j) -                                                 &!hr03
-     &((((((crabamp*crabfreq)*2d0)*pi)/clight)*xv(xory,j))*             &!hr03
-     &cos_rn((((sigmv(j)/clight)*crabfreq)*2d0)*pi + crabph(ix)))*c1m3   !hr03
+       yv(xory,j)=yv(xory,j) - crabamp*c1e3/e0f*                        &!hr03
```

Comment:
I think you need to divide the voltage by 1/(1+delta) in yv update only because yv is Px/P and not Px/P0, while it is correct in the energy.